### PR TITLE
Add benchmark

### LINKED ISSUE #12: Benchmark
To make #11 easier.

### PR DESCRIPTION
**Pull request by C0ldF0x**
**Usage**

```
const benchmark = require('./benchmark');
const stripJsonComments = require('./index');

const json = '{/*rainbows*/"unicorn":"cake"}';

/*
if second parameter is true the result will be printed, otherwise it will be returned as [secs, nanosecs]
*/
benchmark(function () {
    JSON.parse(stripJsonComments(json));
},true);
```

Fixes #12

### CODE REVIEW

**Comment by sindresorhus** [comment]
This is not really a benchmark, as it only tests it once. I would rather do something like https://github.com/sindresorhus/strip-css-comments/commit/3a5ce778272dcf7b7b09b723c201a7679314394e

**Comment by C0ldF0x** [comment]
I was having difficulties getting to the concept of benchmark and this was the first idea I had. Sure, I will do it that way asap. Thanks 👍

**Comment by C0ldF0x** [comment]
Should be as you said now

File: package.json [hunk 1]
```diff
@@ -37,6 +38,7 @@
   ],
   "devDependencies": {
     "ava": "*",
-    "xo": "*"
+    "xo": "*",
+    "matcha":"^0.7.0"
```
Comment (kevva) [inline package.json]: These should be listed in alphabetical order. You could run `npm install --save-dev matcha` and `npm` will put them in the correct order automatically.

File: benchmark.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+const fs = require('fs');
+const stripJsonComments = require('./');
+
+const json = fs.readFileSync('sample.json', 'utf8');
+
+bench('strip JSON comments', function () {
+	stripJsonComments(json);
+});
```
Comment (kevva) [inline benchmark.js]: Missing new line here.

**Comment by sindresorhus** [comment]
Thank you :)

**Pull request merged**